MacVim Fails to build from source on El Capitan

```
CompileC build/MacVim.build/Release/MacVim.build/Objects-normal/x86_64/MMTextViewHelper.o MMTextViewHelper.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
    cd /Volumes/DATA/temp/macvim.git/src/MacVim
    export LANG=en_US.US-ASCII
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x objective-c -arch x86_64 -fmessage-length=120 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -fcolor-diagnostics -Wno-trigraphs -fpascal-strings -Os -mdynamic-no-pic -Wno-missing-field-initializers -Wno-missing-prototypes -Wno-implicit-atomic-properties -Wno-arc-repeated-use-of-weak -Wno-missing-braces -Wparentheses -Wswitch -Wno-unused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wno-empty-body -Wno-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wno-constant-conversion -Wno-int-conversion -Wno-bool-conversion -Wno-enum-conversion -Wno-shorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wno-undeclared-selector -Wno-deprecated-implementations -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk -fasm-blocks -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -mmacosx-version-min=10.11 -fvisibility=hidden -Wno-sign-conversion -I/Volumes/DATA/temp/macvim.git/src/MacVim/build/MacVim.build/Release/MacVim.build/MacVim.hmap -I/Volumes/DATA/temp/macvim.git/src/MacVim/build/Release/include -I/Volumes/DATA/temp/macvim.git/src/MacVim/build/MacVim.build/Release/MacVim.build/DerivedSources/x86_64 -I/Volumes/DATA/temp/macvim.git/src/MacVim/build/MacVim.build/Release/MacVim.build/DerivedSources -F/Volumes/DATA/temp/macvim.git/src/MacVim/build/Release -F/Volumes/DATA/temp/macvim.git/src/MacVim -MMD -MT dependencies -MF /Volumes/DATA/temp/macvim.git/src/MacVim/build/MacVim.build/Release/MacVim.build/Objects-normal/x86_64/MMTextViewHelper.d --serialize-diagnostics /Volumes/DATA/temp/macvim.git/src/MacVim/build/MacVim.build/Release/MacVim.build/Objects-normal/x86_64/MMTextViewHelper.dia -c /Volumes/DATA/temp/macvim.git/src/MacVim/MMTextViewHelper.m -o /Volumes/DATA/temp/macvim.git/src/MacVim/build/MacVim.build/Release/MacVim.build/Objects-normal/x86_64/MMTextViewHelper.o

** BUILD FAILED **


The following build commands failed:
	CompileC build/MacVim.build/Release/MacVim.build/Objects-normal/x86_64/MMApplication.o MMApplication.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)
Makefile:3268: recipe for target 'macvim' failed
make[1]: *** [macvim] Error 65
make[1]: Leaving directory '/Volumes/DATA/temp/macvim.git/src'
Makefile:26: recipe for target 'first' failed
make: *** [first] Error 2
$
```

The problem is in MMApplication.m line 30:

```
    if ((NSEventTypeKeyDown == type || NSEventTypeKeyUp == type) &&
            (flags & NSEventModifierFlagHelp)) {
        flags &= ~NSEventModifierFlagHelp;
```

This fails with message: 
```
/Volumes/DATA/temp/macvim.git/src/MacVim/MMApplication.m:30:10: Use of undeclared identifier 'NSEventTypeKeyDown'; did you mean 'NSEventTypeGesture'?
```

NSEventTypeKeyDown was added in 10.12+ SDK.

--- a/src/MacVim/DBPrefsWindowController.m
+++ b/src/MacVim/DBPrefsWindowController.m
@@ -2,6 +2,7 @@
 //  DBPrefsWindowController.m
 //
 
+#import "MacVim.h"
 #import "DBPrefsWindowController.h"
 
 

--- a/src/MacVim/MMApplication.m
+++ b/src/MacVim/MMApplication.m
@@ -13,6 +13,7 @@
  * Some default NSApplication key input behavior is overridden here.
  */
 
+#import "MacVim.h"
 #import "MMApplication.h"
 
 @implementation MMApplication

--- a/src/MacVim/MMCoreTextView+ToolTip.m
+++ b/src/MacVim/MMCoreTextView+ToolTip.m
@@ -19,6 +19,7 @@
  * changes were made to adapt the code to MacVim.
  */
 
+#import "MacVim.h"
 #import "MMCoreTextView.h"
 
 

